cmd/notation/inspect.go

```diff
@@ -0,0 +1,315 @@
+package main
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-go/plugin/proto"
+	"github.com/notaryproject/notation/internal/cmd"
+	"github.com/notaryproject/notation/internal/envelope"
+	"github.com/notaryproject/notation/internal/ioutil"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/spf13/cobra"
+)
+
+const (
+	treeItemPrefix     = "├──"
+	treeItemPrefixLast = "└──"
+	subTreePrefix      = "│  "
+	subTreePrefixLast  = "   "
+)
```

Comment:
opened issue to track this https://github.com/notaryproject/notation/issues/545

Reply:
If you don't consider printing tree while loading signatures pages, you may consider https://github.com/need-being/go-tree as it is MIT.

Reply:
IMO instead of adding dependency on not a well known library such as `need-being/go-tree` is risky and since the amount of code we are taking about is less than 100 lines I would prefer that we we write our own.